src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/DatabaseErrorPageExtensions.cs

```diff
@@ -26,7 +25,7 @@ public static IApplicationBuilder UseDatabaseErrorPage(this IApplicationBuilder
                 throw new ArgumentNullException(nameof(app));
             }
 
-            return app.UseMiddleware<DatabaseErrorPageMiddleware>();
+            return app.UseDatabaseErrorPage(new DatabaseErrorPageOptions());
```

Comment:
Yep, the migrations end point is a separate piece of middleware. The overload we now delegate to adds both pieces of middleware.